src/NServiceBus.AcceptanceTests/Routing/When_sending_non_message_with_routing_configured_via_mappings.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace NServiceBus.AcceptanceTests.BestPractices
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_sending_non_message_with_routing_configured_via_mappings : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_throw_when_sending()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<Receiver>()
```

Comment:
Do we even need the receiver endpoint, just send the message to your self? (or would we say that the test is more intuitive this way)